app/vmalert/manager.go

```diff
@@ -64,10 +64,11 @@ func (m *manager) startGroup(ctx context.Context, group Group, restore bool) {
 		}
 	}
 
+
 	m.wg.Add(1)
 	id := group.ID()
 	go func() {
-		group.start(ctx, *evaluationInterval, m.storage, m.notifier, m.rw)
+		group.start(ctx, group.Interval, m.storage, m.notifier, m.rw)
```

Comment:
we can omit `group.Interval` in params, since `start` is a method of the `group` object and can refer to the `group.Interval` internally.

Reply:
Fixed

Reply:
@hagen1778